order/src/main/java/com/caldevsupplychain/order/controller/OrderController.java

```diff
@@ -1,8 +1,21 @@
 package com.caldevsupplychain.order.controller;
 
-import com.caldevsupplychain.order.vo.OrderWS;
 import org.springframework.http.ResponseEntity;
 
+import com.caldevsupplychain.order.vo.OrderWS;
+
 public interface OrderController {
+
 	ResponseEntity<?> createOrder(OrderWS orderWS);
+
+	ResponseEntity<?> updateOrder(String orderUuid, OrderWS orderWS);
+
+	ResponseEntity<?> deleteOrder(String orderUuid);
+
+	ResponseEntity<?> readOrder(String orderUuid);
+
+	ResponseEntity<?> readOrdersByUserUuid(String userUuid);
```

Comment:
`readOrdersByUserUuid` and `readOrdersByAgentUuid` can combine to `readOrders(@Nullable String userUuid, @Nullable agentUuid)`. both uuid will be query params in this case.

Reply:
fixed